doc/cephfs/cephfs-shell.rst

```diff
@@ -14,6 +14,17 @@ Options :
     -b, --batch FILE      Process a batch file.
     -t, --test FILE       Test against transcript(s) in FILE
 
+
+.. note::
+
+    For running cephfs-shell, at least one metadata pool and one data pool should be active.
+    If CephFS installed through source, execute cephfs-shell in the build directory.
+    It can also be executed as following:
+
+.. code:: bash
+
```

Comment:
add a note to source the new `vstart_environment.sh` file:

$ (source vstart_environment.sh && python3 ../src/tools/cephfs/cephfs-shell)

See also #27198